### ISSUE TITLE
prep mini laser A5

### ISSUE THREAD

**Issue by nicolasdb** [nicolasdb/nicolasdb.github.io#66]
@DewiBrunet

j'ai repris la suite.
 
## assembly
haha!
il restait pas mal de pieces non utilisee, it's always a bad sign of a well done job. ;)
so the missing motor as we talked, is actually compensated by a rod an another pulley+belt on the other side
![image](https://user-images.githubusercontent.com/12049360/36902358-81bce888-1e2a-11e8-8e1e-96bf1502c097.png)

### download and install has explain here:
https://www.laseraxe-customize.com/forum/cncc-software/cncc-v2-53c-released-and-support-dxf-file

**Comment by nicolasdb** [comment]
### software 
#### laserweb
https://www.laseraxe-customize.com/forum/laseraxe-controller-board/running-laserweb
https://github.com/1wise/grblv1.1-laseraxe

#### origin
for the time being, i'll follow what they propose

**Comment by nicolasdb** [comment]
D'apres les conseils de Julien, faire les reglages grbl avec bcnc
https://github.com/vlachoudis/bCNC
ensuite laserweb

**Comment by nicolasdb** [comment]
bCNC sur windows
https://github.com/vlachoudis/bCNC/wiki/OS

okokok, bCNC en ligne dans windows>>>> mais j'arrive a rien
back to school avec laserweb

good, some resultat avec 
https://github.com/LaserWeb/LaserWeb4/wiki/2.1-Initial-Configuration-GRBL-1.1
et ensuite ceci:
https://github.com/gnea/grbl/blob/master/doc/markdown/settings.md

problème de direction Y, résolus avec la commande 
`$3=2`

**Comment by nicolasdb** [comment]
## yeassss 
ok, it's working!!!
so, we need to add in laserweb, settings\gcode panel
tool on: M3
tool off: M5
Power max 255

pour le test c'est 
tool test power 50
tool test durqtion 1000

**Comment by nicolasdb** [comment]
![image](https://user-images.githubusercontent.com/12049360/36992660-e7882172-20ab-11e8-9df6-0aa88c852093.png)

**Comment by nicolasdb** [comment]
okok, problème de calibration XY. 
un dessin de 100mm, résulte en une ligne de 80mm. 
### check step/mm
https://learn.sparkfun.com/tutorials/stepoko-powered-by-grbl-hookup-guide/firmware-configuring-grbl-and-calibrating

 $100=80.000
 $101=80.000
-> 100mm=80mm
**Soit**
![image](https://user-images.githubusercontent.com/12049360/37532575-ae9e408e-293f-11e8-994a-9953b784f8de.png)

**donc**
80*(100/80)= 100
_obviously...._

Set to 100 step/mm
```
$100=100
$101=100
```

~~pour Y, légère différence. 
100=99mm
100*(100/99) = 101~~

**Comment by nicolasdb** [comment]
## test protocole d'utilisation:
- installer laserweb
- config ok? oui/non, si non, voir maintenance
- import svg from inkscape.
![image](https://user-images.githubusercontent.com/12049360/38041622-88417a6a-32b2-11e8-8733-96f8b553af6b.png)
- drag and drop le fichier en question dans la zone grise plus bas
![image](https://user-images.githubusercontent.com/12049360/38041743-d4f82232-32b2-11e8-89d8-acebdb624ec3.png)
- définis tes réglages en fonction du job. (todo: préparer une série de présélection)
![screenshot 85](https://user-images.githubusercontent.com/12049360/38041899-3ee17cca-32b3-11e8-9422-7314a94d44fe.jpg)
Click sur "generate" l'onglet au dessus.

- connection à la machine
![image](https://user-images.githubusercontent.com/12049360/38042026-860c6218-32b3-11e8-93e6-bf2ed733d773.png)

- set zero et controle manuel. 
![image](https://user-images.githubusercontent.com/12049360/38042064-9aee6a64-32b3-11e8-9cef-759672873992.png)

- run the job.
merci bonsoir.

**Comment by nicolasdb** [comment]
- découpe extérieur
![image](https://user-images.githubusercontent.com/12049360/38042891-d6e9b814-32b5-11e8-94ca-27e1f79e8407.png)
- on place le badge découpé dans lasersaur
![image](https://user-images.githubusercontent.com/12049360/38042932-f921c250-32b5-11e8-82dd-d5e396e332da.png)
- on lui met sa race à coup de laser, pioupiou
![image](https://user-images.githubusercontent.com/12049360/38042971-0e657198-32b6-11e8-906f-6e8259917ba7.png)

- tadaaaa
![image](https://user-images.githubusercontent.com/12049360/38043009-23e055c4-32b6-11e8-94ae-7d999f93ec11.png)

**Comment by nicolasdb** [comment]
ok, y a eu un bug .... :D

**Comment by nicolasdb** [comment]
![image](https://user-images.githubusercontent.com/12049360/38044474-3017ec18-32ba-11e8-879c-bc07e522b887.png)

à nouveau, un bug, la machine s'arrête mais pas le laser, curieux. Faut que j'enquête.

**Comment by nicolasdb** [comment]
https://www.thingiverse.com/thing:2636628
J'ai sauvegardé les éléments structurel de l'elekmaker, c'est similaire. Je pense que le chariot du laser, l'axe X, mériterai d'être revu.

**Comment by milenasonneveld** [comment]
![img_2654](https://user-images.githubusercontent.com/31619215/44517070-fccf9c80-a6c6-11e8-825e-116b260d86f7.JPG)
L’origine de la machine. Mettez la tête au (0,0), bougez-le mécaniquement quand elle est éteinte.
![machine set zero](https://user-images.githubusercontent.com/31619215/44517161-37d1d000-a6c7-11e8-9ac2-92debf41aa35.PNG)
Puis dans laserweb, cliquez sur 'set zero' pour mémoriser l'origine dans le logiciel.

Pour info, les dimensions de la machine sont (x,y) = (width 170mm height ,200mm). Cela s'introduit dans 'Settings'.
![machine deimensions](https://user-images.githubusercontent.com/31619215/44517234-6d76b900-a6c7-11e8-9733-03c50d1e6e72.PNG)

**Comment by milenasonneveld** [comment]
! Parfois Laserweb fait planter les ordinateurs Windows. 
Utilisez Linux pour éviter ce problème.

**Comment by nicolasdb** [comment]
yup, gros BSOD windows à cause dun bug au niveau de la puce USB de l'arduino + laserweb que dans windows. J'ai testé avec le 8 et le 10. même chose. 
C'est stable sous linux.
